src/Server/ServerAuthorization.luau

```diff
@@ -85,51 +96,83 @@ end
 Initializes a player.
 --]]
 function ServerAuthorization.InitializePlayer(self: ServerAuthorization, Player: Player): ()
+    if self.GroupAdminLevelCache[Player.UserId] then
+        self.GroupAdminLevelCache[Player.UserId] = nil
+    end
```

Comment:
This can be simplified to just `self.GroupAdminLevelCache[Player.UserId] = nil`.